docker/package/scripts/update-binaries-list.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import os
+import sys
+import json
+
+binaries_json_path_suffix = "/tests/binaries.json"
+binaries_list_path_suffix = "/tmp/binaries.txt"
+
+
+def update_binaries(binaries, field):
+    binaries_json_path = os.environ["PWD"] + binaries_json_path_suffix
+    with open(binaries_json_path, "r") as file:
+        data = json.load(file)
+
+    data[field] = binaries
+    with open(binaries_json_path, "w") as file:
+        json.dump(data, file, indent=4)
+
+
+def main():
+    tag = os.environ["BUILDKITE_TAG"]
+    binaries = []
+
+    binaries_list_path = os.environ["PWD"] + binaries_list_path_suffix
+    with open(binaries_list_path, "r") as f:
+        binaries = [l.strip() for l in f.readlines()]
```

Comment:
It seems that we should do `s/octez/tezos/` for binary names as well.

The reason is that tests in `docker/tests-test-*-binaries.py` install **packages** using `apt` and `rpm`, and the package names are `tezos-*`, not `octez-*`, so that `apt install octez-client` will fail, and the right command is `apt install tezos-client`.

For more context about binaries/packages naming policy see [this thread](https://serokell.slack.com/archives/C013LJZVC1K/p1665650088834379?thread_ts=1665502772.172429&cid=C013LJZVC1K)